girder/utility/plugin_utilities.py

```diff
@@ -35,11 +35,14 @@
 import sys
 import traceback
 import yaml
+import importlib
+
+from pkg_resources import iter_entry_points
 
 from girder.constants import PACKAGE_DIR, ROOT_DIR, ROOT_PLUGINS_PACKAGE, \
     TerminalColor
 from girder.models.model_base import ValidationException
-from girder.utility import config, mail_utils, mkdir
+from girder.utility import config as _config, mail_utils, mkdir
```

Comment:
In girder/\__init\__.py, we can just add `import girder.utility.plugin_utilities as plugin`, then we would have the namespace you want as an alias. I certainly like that name better.

While I agree that a decorator should normally describe the wrapped function, the function in question is actually not a decorator, it's a function that takes some arguments and returns a decorator. That might be splitting hairs, but to me the passing of parameters is an important distinction that makes me believe it's more useful to describe the decorator-returning function than the wrapped function. So, I lean toward `girder.plugin.config` over `girder.plugin.loader`, but we should decide democratically. @girder/developers